tsconfig.json

```diff
@@ -11,5 +11,6 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "sourceMap": true
-  }
+  },
+  "exclude": ["test/**"]
```

Comment:
Do not exclude `test` folder otherwise TS compiler acts weirdly and you had to add those `unknown` casts to the tests which is not ideal. Maybe better way is to remove this line and configure Jest that it only runs those ts files 👍